app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -16,6 +16,12 @@ def institution_name_full
       t('sufia.institution_name_full', default: institution_name)
     end
 
+    # Which translations are available for the user to select
+    # @return [Hash<String,String>] locale abbreviations as keys and flags as values
+    def available_translations
+      { 'en' => '🇬🇧', 'es' => '🇪🇸' }
```

Comment:
I like the way the WDL site approaches this: https://www.wdl.org/en/item/1/# (See the flag dropdown.)